### ISSUE TITLE
Geburtstagsmodul- Modus Bootstrap und Jssor werden nicht richtig angezeigt

### ISSUE THREAD

**Issue by svdoldie** [diddipoeler/sportsmanagement#1169]
Ich wollte im Geburtstagmodul mal die beiden Modi Bootstrap und Jssor ausprobieren, allerdings konnte ich beide nicht so konfigurieren, dass die Ansicht ordentlich ist. Im Modus Bootstrap werden die Personen untereinander angezeigt und nicht einzelnd. Im Modus Jssor werden die Personenbilder nicht richtig angezeigt

**Comment by diddipoeler** [comment]
Joomla 3 oder/und 4 ?

**Comment by svdoldie** [comment]
Joomla 3 und 4

**Comment by diddipoeler** [comment]
Bei mir funktinniert das.

**Comment by svdoldie** [comment]
Bei mir leider nicht, und auch nicht auf deiner Joomla 4 Testseite

**Comment by diddipoeler** [comment]
Jetzt besser ?

**Comment by svdoldie** [comment]
Nein. Im Modus Bootstrap werden alle Geburtstage untereinander angezeigt.
Die Navigationspfeile haben keine Wirkung.
Der Modus Jssor Slider hat Probleme mit der Darstellung, wenn mehrere Personen denselben Geburtstag haben.
Siehe https://www.svdoerpum.de/
Eingestellt ist der Jssor Slider

**Comment by svdoldie** [comment]
Kann man im Jssor-Modus entweder den Text oder das Bild nach rechts verschieben, so das beides nicht übereinandergelegt angezeigt wird?

**Comment by diddipoeler** [comment]
Bitte noch einmal mit Bootstrap testen.

**Comment by diddipoeler** [comment]
Bitte noch einmal mit Bootstrap testen.

**Comment by svdoldie** [comment]
Die Breite ist jetzt bei allen gleich, nur die eingestellte Höhe scheint noch nicht übernommen zu werden

**Comment by svdoldie** [comment]
Irgendwie zerschießt das Modul jetzt aber mein Template, siehe links weiter oben
